### DISCUSSION TITLE
BVH 部分很多错误翻译

### DISCUSSION THREAD

**Discussion by LiuZengqiang** [kanition/pbrtbook#149] [交流]
BVH、HLBVH 部分很多错误翻译，感觉像是简单的机翻，甚至连人工检查都没有。

**Comment by kanition** [comment]
欢迎为本书错误提供改进意见和建议！您可以参照issue模板提出具体改进信息。

本人再次郑重声明，全书文本都经本人之手翻译。

本人作为图形学初学者，英语水平也不优秀，虽追求能力范围之内翻到最好，但遇到不会的地方是常有的事。不少难以翻译的地方，我都会通过翻译软件查询词义疏通大意，然后结合本人理解并斟酌后撰写，以避免语言问题造成技术原理上的错误解释。

为了保留原书风格，我只在有限程度内拆分长难句，没有大刀阔斧地改写为短句。这也是本书读起来有点机翻风格的最大原因。但原书就是如此写作的，本人还没有水平在语言风格层级上对原书内容进行调整。

然而这绝不意味着本书存在不加校对直接复制粘贴机翻结果的情况。一句都没有！本书也不是整个篇幅送进翻译软件后，再对个别词句进行调整。而是我读得懂的都我自己翻，读不懂的查资料或用翻译软件理解后再自己翻。基础文本一定是本人输出的。

序言中我阐述了对“作品”的追求，也许本人水平有限导致质量不高，但请放心我对翻译本书的态度。我相信补充章节和批注可以说明这一点。

**Comment by LiuZengqiang** [comment]
是我大意了
